store/sql_user_store.go

```diff
@@ -1097,8 +1099,9 @@ func (us SqlUserStore) Search(teamId string, term string, searchType string) Sto
 			FROM
 				Users
 			WHERE
-				DeleteAt = 0
+				Id != ''
```

Comment:
0/5 maybe we should add a note to the code, otherwise during a refactor I would probably delete that :)